src/components/Layouts.jsx/MainLayout.jsx

```
@@ -400,198 +401,296 @@ const MainLayout = (props) => {
+                  startShowCase && (
+                    <Box
+                      position={"absolute"}
+                      top={20}
+                      left={20}
+                      justifyContent={"center"}
+                    >
+                      <Box display={"flex"}>
+                        {[
+                          ...Array(Math.max(0, redLivesToShow) || 0).keys(),
+                        ]?.map((elem) => (
                           <Diamond />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)